src/js/database.js

```diff
@@ -290,6 +290,9 @@ function DataManager(db, resourceDir, sourceDir, configurator) {
                 })
 				.then(function () {
                     return mythis.activateContainer(language, "bible", "tw");
+                })
+                .catch(function (err) {
```

Comment:
Does the catch() function stop further processing?  For example, if "tq" is not present, then will this prevent teh code from attempting to load "udb" and "tw"?

Reply:
That's a really good point. You are right, other resources won't be activated if English TQ threw an exception before them. So I moved `catch` construct inside `activateContainer` method to gracefully skip activation of bad/absent resources.